src/libConsensus/ConsensusBackup.cpp

```diff
@@ -47,6 +47,7 @@ bool ConsensusBackup::CheckState(Action action) {
                                       // get ignored by the node)
        {FINALCOMMIT_DONE, PROCESS_FINALCHALLENGE},
        {FINALCOMMIT_DONE, PROCESS_FINALCOLLECTIVESIG},
+       {FINALRESPONSE_DONE, PROCESS_FINALCHALLENGE},
```

Comment:
Can you confirm that this action is really allowed in this state?

Reply:
Yes, that’s because we need to process multiple challenges inspite of responding to first challenge received.. otherwise given backup will send response only to first subset challenge and ignore other subset’s challenge.. i had tested this scenario.